data/config/court.mainnet.js

```diff
@@ -10,13 +10,13 @@ const START_DATE = Math.floor(new Date('2020-02-10T00:00:00.000Z') / 1000)
 const ANJ = {
   symbol: 'ANJ',
   decimals: 18,
-  address: '0xcD62b1C403fa761BAadFC74C525ce2B51780b184'          // AGP-126
+  address: require('../output/minime.mainnet').ANJ.address
```

Comment:
My only concern here is that it's weird to use the `output` dir in a `config` file.

I guess for networks other than Mainnet, this is unavoidable because we deploy those tokens as part of the deployment process?

Reply:
Yes it is true, but we were repeating tons of addresses in different places. We were messing up with them and it was hard to keep all those places up-to-date. I implemented a "[lazy require](https://github.com/aragon/aragon-network-deploy/pull/62)" to log in case there is no output file when requiring things like this. 

However, note that for all the networks, if you first deploy the tokens and then the court, you won't face any issues. It's just a matter that you need to run the commands in order (which makes sense) or have an output file indicating the addresses you want to use

Reply:
Saw https://github.com/aragon/aragon-network-deploy/pull/62 afterwards and that makes sense!